pulpcore/pulpcore/tasking/services/worker_watcher.py

```diff
@@ -76,17 +77,20 @@ def check_celery_processes():
     now = timezone.now()
     oldest_heartbeat_time = now - timedelta(seconds=TASKING_CONSTANTS.PULP_PROCESS_TIMEOUT_INTERVAL)
 
-    for worker in Worker.objects.filter(last_heartbeat__lt=oldest_heartbeat_time, online=True):
+    for worker in Worker.objects.filter(last_heartbeat__lt=oldest_heartbeat_time,
```

Comment:
I think this should switch to using the `workers.online_workers()` manager queryset method. It seems to do the same thing. Is that right?

Reply:
It's actually doing precisely the opposite.  It's getting workers that have timed out but haven't had ```gracefully_stopped``` set to ```True```, e.g. crashed/missing workers.